src/Transformers/TransformerItem.php

```diff
@@ -9,29 +9,17 @@
 
 class TransformerItem
 {
-    protected $resource;
+    protected TransformerInterface $transformer;
 
-    /** @var TransformerInterface */
-    protected $transformer;
-
-    /** @var TransformerFactory */
-    protected $transformerFactory;
-
-    /**
-     * @param TransformerFactory $transformerFactory
-     * @param mixed              $resource
-     */
-    public function __construct(TransformerFactory $transformerFactory, $resource)
-    {
-        $this->resource = $resource;
-        $this->transformerFactory = $transformerFactory;
+    public function __construct(
+        protected TransformerFactory $transformerFactory,
+        protected mixed $resource,
```

Comment:
Original was nullable
```suggestion
        protected ?mixed $resource,
```

Reply:
Nullable as in `null` could be passed in the constructor, but this is still possible. Both the old and new situation require a `$resource` as second parameter, otherwise the constructor throws an `ArgumentCountError`.